src/components/Sheet/README.md

```diff
@@ -207,9 +207,11 @@ class SheetExample extends React.Component {
                   </div>
                   <Scrollable style={{padding: '1.6rem', height: '100%'}}>
                     <ChoiceList
+                      title="Select a sales channel"
```

Comment:
This is good. Making a note that it also caused me to find an unnecessary and invalid `aria-describedby` attribute on the `<fieldset>` output. Will write that up.

Reply:
New issue: #1589